stories/module-analytics-components.stories.js

```diff
@@ -50,46 +51,67 @@ import {
 	dashboardUniqueVisitorsVisitorData,
 } from '../assets/js/modules/analytics/datastore/__fixtures__';
 
+/**
+ * Defines some additional setup for all tests.
+ *
+ * @since n.e.x.t
+ *
+ * @param {wp.data.registry} registry Registry with all available stores registered.
+ */
+const setup = ( registry ) => {
```

Comment:
A similar `setup` function should be added for all the Search Console components stories, which sets the Search Console `propertyID` setting (see comment further above), otherwise the URLs there are incorrect (omitting `property_id`).